packages/wheelhouse-frontend/src/handlers/component-handlers/log-handlers.js

```diff
@@ -0,0 +1,34 @@
+
+export const hashCode = (str) => { 
+  var hash = 0;
+  for (var i = 0; i < str.length; i++) {
+    hash = str.charCodeAt(i) + ((hash << 5) - hash);
+  }
+  return hash;
+}; 
+
+export const _makeBrighter = (hex) => {
+  hex = hex.split(""); 
+  hex[1] = "F"; 
+  return hex.join("");
+};
+
+export const intToRGB = (i) => {
```

Comment:
Love this! But let's move it to `wheelhouse-core` so we can use the same algorithm to print out the colors of the app in the CLI.